code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -108,6 +108,32 @@
 				On_Consume(M)
 			return 1
 
+	if (istype(M, /mob/living/simple_animal))

```

Comment:
Maybe, but i'm not using it because (without looking yet) there are only two possibilities:

1. the isAnimal proc just checks if something is a simple animal and returns a bool. Worthless and just adds overhead

2. the isAnimal proc checks for some other things beside simple animals, then my code throws a runtime error trying to access a variable only simple animals have

i have something else to add to this, so ill go back and check anyway, but odds are the helper isnt right for this situation even if it exists